src/utils/viewer/ObjectPickingHelper.cpp

```diff
@@ -0,0 +1,132 @@
+#include "ObjectPickingHelper.h"
+#include <Corrade/Containers/StridedArrayView.h>
+#include <Corrade/Utility/Assert.h>
+#include <Magnum/GL/Framebuffer.h>
+#include <Magnum/GL/Renderbuffer.h>
+#include <Magnum/GL/RenderbufferFormat.h>
+#include <Magnum/Image.h>
+#include <Magnum/Magnum.h>
+#include <Magnum/PixelFormat.h>
+#include <Magnum/Shaders/Generic.h>
+#include <Magnum/Shaders/MeshVisualizer.h>
+
+namespace Cr = Corrade;
+namespace Mn = Magnum;
+using Mn::Math::Literals::operator""_rgbf;
+
+ObjectPickingHelper::ObjectPickingHelper(Mn::Vector2i viewportSize) {
+  // create the framebuffer and set the color attachment
+  recreateFramebuffer(viewportSize).mapForDraw();
+  CORRADE_INTERNAL_ASSERT(
+      selectionFramebuffer_.checkStatus(Mn::GL::FramebufferTarget::Draw) ==
+      Mn::GL::Framebuffer::Status::Complete);
+
+  // create a shader
+  Mn::Shaders::MeshVisualizer3D::Flags flags =
+      Mn::Shaders::MeshVisualizer3D::Flag::Wireframe;
+  shader_ = std::make_unique<Mn::Shaders::MeshVisualizer3D>(flags);
+
+  shader_->setViewportSize(Mn::Vector2{viewportSize});
+
+  if (flags & Mn::Shaders::MeshVisualizer3D::Flag::Wireframe) {
+    (*shader_).setColor(0x2f83cc_rgbf).setWireframeColor(0xdcdcdc_rgbf);
```

Comment:
@mosra: Should I set the alpha for the color of the MeshVisualizer3D here??

Reply:
Yep, `0x2f83cc7f_rgbaf` for example.